### PR TITLE
fix 5 parse function

### PR DESCRIPTION
**Pull request by lyc200264-stack**
## PR Type
[Bug Fix] Fix 5 parse functions in multi_swe_bench

## PR Title

[Bug Fix] Correct parsing logic for 5 functions in multi_swe_bench

## Summary

This PR fixes 5 parsing functions in `multi_swe_bench` that were producing incorrect
outputs or errors in certain edge cases. The changes touch 5 files, each related to the
respective parsing logic.

## Motivation

These fixes are necessary to ensure that the parsing logic works correctly across
different input formats and edge cases, preventing downstream errors in evaluation.

## Testing

- All affected parsing functions were tested locally with unit tests.
- Existing tests pass successfully.

### CODE REVIEW

File: multi_swe_bench/harness/repos/javascript/Seneca_CDOT/telescope_3768_to_3431.py [hunk 1]
```diff
@@ -180,22 +178,43 @@ def fix_patch_run(self, fix_patch_run_cmd: str = "") -> str:
...
-        # Identify services with failed tests via summary lines (allow any characters between lines)
-        failed_services_pattern = re.compile(r'@senecacdot/([^:]+):test:\s+Test Suites:\s+\d+ failed, \d+ total.*?@senecacdot/\1:test:\s+Tests:\s+(\d+) failed', re.DOTALL)
-        failed_services = failed_services_pattern.findall(log)
-        for service, _ in failed_services:
-            # Extract all test names (including both passed and failed) for the service
-            service_tests_pattern = re.compile(rf'@senecacdot/{service}:test:\s+[✓✕]\s+(.+?)\s+\(\d+ ms\)')
-            all_tests = set(service_tests_pattern.findall(log))
-            # Failed tests = total tests in service - passed tests
-            failed_tests.update(all_tests - passed_tests)
+        #修改前 # Parse passed tests
```
Comment (daoguang-coder) [inline multi_swe_bench/harness/repos/javascript/Seneca_CDOT/telescope_3768_to_3431.py]: 这些都去掉
